.gitignore

```diff
@@ -15,7 +15,3 @@ distribution/smarthome/runtime/etc/mapdb/storage.mapdb.t
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb.p
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb.t
-bundles/model/antlr-generator-3.2.0-patch.jar
```

Comment:
Has the location of antlr-generator and the storage.mapdb been changed by the Xtext update?